Resources/Locale/en-US/accent/german.ftl

```diff
@@ -0,0 +1,203 @@
+accent-german-words-1 = yes
+accent-german-words-1-2 = yea
+accent-german-words-replace-1 = ja
+
+accent-german-words-2 = no
+accent-german-words-replace-2 = nein
+
+accent-german-words-3 = the
+accent-german-words-replace-3 = ze
```

Comment:
ze sounds like "ce" (c like in center) do i suggest to change this replacement to `de`

Reply:
It's not exactly like in center, but a main part of a strong German accent is the 'th' turning into more of an s/z sound cause 'th' isn't a thing in German and 's' is the closest to it.

The french accent already does this (though it does ``the`` => ``'ze``) and this is pretty much the same thing.

This'd also need to replace "that", "this" and so on with "dat" and "dis" which, to me, reads as internet silly speak.

Reply:
It's an accent, `ze` isn't meant to be read in German but in English. It's how a typical German accent sounds like when they try to pronounce `the`. Ex: see [TF2 Medic](https://wiki.teamfortress.com/wiki/Medic_voice_commands#Voice_Menu_2)

Reply:
> also `the` is `der/die/das` in german so its really better

It already turns like 30% of all ``the`` into ``das``, before the replacements.
I think doing more would get a bit much and I think doing only ``das`` just makes it funnier since the gender won't match anyways and ``das`` feels like *the* German article. (also "die" might lead to odd sentenced)